tests/test_corona.py

```diff
@@ -278,16 +278,30 @@ def test_pad_cumulative_series_to_yesterday():
 
 def test_germany_get_population():
     germany = c.germany_get_population()
-    assert germany.shape == (412, 38)
+
+    assert germany.index.name == 'county'
+    assert 'population' in germany.columns
+    assert 'cases7_per_100k' in germany.columns
```

Comment:
This one looks a bit fragile, and we don't use it as far as I remember. Therefore I admit we might remove this check. But the current way if fine for me.

Reply:
Got a request through to use the RKI-provided 7-day numbers instead of our 14-day numbers that we calculate, I haven't gotten around to doing this yet, but it'll be used once I do.